xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-api/src/main/java/org/xwiki/notifications/filters/internal/scope/ScopeNotificationFilterPreference.java

```diff
@@ -248,4 +249,10 @@ public int hashCode()
             .append(children)
             .toHashCode();
     }
+
+    @Override
+    public String toString()
+    {
+        return ToStringBuilder.reflectionToString(this);
```

Comment:
Not a big deal but we have a custom implementation for the toString: https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HEquals2FHashCodeandToStringimplementations.

Reply:
Ok, I have updated both ``toString()`` methods